64_FFT/app_resources/shader.comp.hlsl

```diff
@@ -0,0 +1,79 @@
+#include "common.hlsl"
+#include "nbl/builtin/hlsl/workgroup/fft.hlsl"
+//#include "nbl/builtin/hlsl/subgroup/fft.hlsl"
+#include "nbl/builtin/hlsl/glsl_compat/subgroup_basic.hlsl"
+
+[[vk::push_constant]] PushConstantData pushConstants;
+
+groupshared output_t sharedmem[2 * WorkgroupSize];
+
+struct SharedMemoryAccessor {
+	void set(uint32_t idx, output_t x) {
+		sharedmem[idx] = x;
+	}
+	
+	output_t get(uint32_t idx) {
+		return sharedmem[idx];
+	}
+
+	void workgroupExecutionAndMemoryBarrier() {
+        AllMemoryBarrierWithGroupSync();
+    }
```

Comment:
indentation, also I'd rather have you use the `glsl::` functions than HLSL intrinsics

Reply:
reminder